scipy/linalg/tests/test_basic.py

```diff
@@ -730,9 +730,9 @@ def test_multiple_rhs(self):
 
     def test_transposed_keyword(self):
         A = np.arange(9).reshape(3, 3) + 1
-        x = solve(np.tril(A)/9, np.ones(3), transposed=1)
+        x = solve(np.tril(A)/9, np.ones(3), transposed=True)
         assert_array_almost_equal(x, [1.2, 0.2, 1])
-        x = solve(np.tril(A)/9, np.ones(3), transposed=0)
+        x = solve(np.tril(A)/9, np.ones(3))
```

Comment:
This test should not be changed --- should be left as is, so the transposed gets tested.